solidity/nomad-xapps/contracts/bridge/BridgeRouter.sol

```diff
@@ -377,6 +383,23 @@ contract BridgeRouter is Version0, Router {
             PRE_FILL_FEE_DENOMINATOR;
     }
 
+    /**
+     * @notice Dust the recipient.
+     * @dev Does not dust if insufficient funds, or if user has funds already
+     */
+    function _dust(address _recipient) internal {
+        if (
+            _recipient.balance <= DUST_AMOUNT &&
+            address(this).balance >= DUST_AMOUNT
+        ) {
+            // `send` gives execution 2300 gas and returns a `success` boolean.
```

Comment:
we could send the amount to top-up to DUST_AMOUNT every time, but eh, bit more complex, not bothered to worry about it